### ISSUE TITLE
Plugin dependency loading fails

### ISSUE THREAD

**Issue by ssteiner** [natemcmaster/DotNetCorePlugins#125]
I've been looking for a solution to use my .NET Std 2.0 plugins from a .NET Core app and found your lib. With it, I managed to instantiate my IPlugins. But, when I then call into a plugin method, I get another "Could not load file or assembly ..."exception for a dependency of my plugin. The dll it cannot load is right there in the plugin folder, but I guess the resolve only happens when I call into code that uses the dependency. And since that's not in the context of your PluginLoader, it craps out (as would my original plugin loading code that works fine on .NET full).

What do I need to do to be able to use plugins that load dependencies themselves (there's no dependency loading in the plugin itself.. it's just that the namespace that is in the dependency of the plugin is not used during instantiation, but is used when I call a method on the plugin in interface).

**Comment by natemcmaster** [comment]
It's hard to say. Can you provide steps to reproduce the problem? A minimal repro is the ideal way for me to see what is going on in your project.

**Comment by ssteiner** [comment]
I've managed to resolve the issue. In the end, it was an output configuration issue. I have one plugin that talks to a native 32bit DLL, so the plugin is compiled for x86. When I directly compile the plugin into a .NET core app, that app needs to be compiled for x86, too. My app that loads the plugins was set to AnyCPU. Once I changed it to x86, it worked. I did not immediately catch that because I wasn't getting the InvalidFormatException you usually get when mixing output.
